client/src/containers/app.coffee

```diff
@@ -17,10 +18,11 @@ App = React.createClass
     loginActions = bindActionCreators LoginActions, dispatch
     pageLocationActions = bindActionCreators PageLocationActions, dispatch
     repoActions = bindActionCreators RepoActions, dispatch
+    registerActions = bindActionCreators RegisterActions, dispatch
 
     div {},
       Header {login, loginActions, pageLocationActions}
-      MainSection {login, repo, repoActions, pageLocation, loginActions, pageLocationActions}
+      MainSection {login, repo, repoActions, pageLocation, loginActions, pageLocationActions, registerActions}
```

Comment:
As far as I can tell, the canonical solution to this problem in Redux is to move the component "closer" to redux. Make the component know directly about redux via connect. This needs research though. It may be that _all_ main components are _encouraged_ to know about the store via `connect` and that just page widgets should be in the dark. Not sure what the best pattern is here.